examples/main/layer-controls.js

```diff
@@ -38,7 +38,8 @@ export default class LayerControls extends PureComponent {
 
   _renderSlider(settingName, value) {
     let max = 1;
-    if (/radius|width|height|pixel|size/i.test(settingName)) {
+    if (/radius|width|height|pixel|size/i.test(settingName) &&
+      !(/^((?!scale).)*$/).test(settingName)) {
```

Comment:
These stacked regular expressions are not exactly easy to read, especially if more cases are added.

Some relief could come from defining them in well-named constants, or through creating a helper function, or at minimum adding a comment.